test/unit/testAddAlias.js

```diff
@@ -4,7 +4,7 @@ const initSteps = require('app/core/initSteps');
 const assert = require('chai').assert;
 
 describe('AddAlias', () => {
-    const steps = initSteps([`${__dirname}'/../../app/steps/action/`, `${__dirname}/../../app/steps/ui`]);
+    const steps = initSteps([`${__dirname}/../../app/steps/action/`, `${__dirname}/../../app/steps/ui/`]);
```

Comment:
Additional '/' added to app/steps/ui/   - I think this isn't required - please confirm